draft-packages/stories/Dropdown.stories.tsx

```diff
@@ -62,6 +62,14 @@ const Menu: React.FunctionComponent = () => (
 
```

Comment:
I think this file should instead be in the component library stories.. I'm a little confused about this one. Looks like the React component is a core component, but the Elm components are draft.

Reply:
Ah, this rings a bell - I remember coming across a component that was core in React and draft in Elm (or vice versa)